spec/capitalize_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './capitalize_decorator'
+require './nameable'
+describe CapitalizeDecorator do
```

Comment:
- [ Required ] Please consider adding a unit test for the `correct_name` method in the `CapitalizeDecorator` class. One of the project requirements is testing all the methods in our classes including those created while implementing the Decorator design pattern. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-06-16 18-38-05](https://github.com/yosaddis/School-library-Ruby/assets/16392046/d02c087f-c9ca-494b-bdf0-7a7a651ebd61)